src/App.js

```diff
@@ -21,6 +21,7 @@ class App extends Component {
           <Route exact path="/" component={crowdsaleAddr?Crowdsale:Home}/>
           <Route exact path="/crowdsale" component={Crowdsale}/>
           <Route exact path="/invest" component={Invest}/>
+          <Route exact path="/crowdsaleDetails" component={Contracts}/>
```

Comment:
Can we use another path so that we don't confuse it with `/crowdsale`? Maybe `manageCrowdsale` or just `manage`?

Reply:
I'll change it to `manage` as it seems the most accurate for the current routes naming